src/test/kotlin/org/rust/lang/core/type/RsTypeResolvingTest.kt

```diff
@@ -510,7 +510,7 @@ class RsTypeResolvingTest : RsTypificationTestBase() {
 
         val ty = typeAtCaret.type
         val renderedTy = when (renderMode) {
-            DEFAULT -> ty.toString()
+            DEFAULT -> ty.render(useAliasNames = false)
```

Comment:
i think skipUnchangedDefaultTypeArguments` should also be `false` (as well as in `WITH_LIFETIMES` and `WITH_ALIAS_NAMES` cases) to be uniform with `RsTypificationTestBase`